flak test in acquaintance/gates_test.py

```
There` is a flaky test in acquait gates.

_____________________ test_acquaint_part_pairs[part_lens2] _____________________
part_lens = (3, 3, 3, 3, 3, 3, ...)
    @pytest.mark.parametrize(u'part_lens',
        [tuple(randint(1, 3) for _ in xrange(randint(2, 10))) for _ in xrange(3)])
    def test_acquaint_part_pairs(part_lens):
        parts = []
        n_qubits = 0
        for part_len in part_lens:
            parts.append(tuple(xrange(n_qubits, n_qubits + part_len)))
            n_qubits += part_len
        qubits = tuple(cirq.NamedQubit(s) for s in alphabet)[:n_qubits]
        swap_network_op = cca.SwapNetworkGate(
>           part_lens, acquaintance_size=None)(*qubits)
cirq/contrib/acquaintance/gates_test.py:144: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
cirq/ops/raw_types.py:133: in __call__
    return self.on(*args, **kwargs)
cirq/ops/raw_types.py:129: in on
    self.validate_args(qubits)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = cirq.contrib.acquaintance.SwapNetworkGate((3, 3, 3, 3, 3, 3, 3, 3, 3, 2), None)
qubits = (cirq.NamedQubit('a'), cirq.NamedQubit('b'), cirq.NamedQubit('c'), cirq.NamedQubit('d'), cirq.NamedQubit('e'), cirq.NamedQubit('f'), ...)
    def validate_args(self, qubits):
        if len(qubits) != self.num_qubits():
[1m            raise ValueError(
                u'{}-qubit gate was applied to {} qubits'.
>                   format(self.num_qubits(), len(qubits)))
E           ValueError: 29-qubit gate was applied to 26 qubits
```

--- a/cirq/contrib/acquaintance/gates_test.py
+++ b/cirq/contrib/acquaintance/gates_test.py
@@ -135,7 +135,7 @@ def test_acquaint_part_pairs(part_lens):
     for part_len in part_lens:
         parts.append(tuple(range(n_qubits, n_qubits + part_len)))
         n_qubits += part_len
-    qubits = tuple(cirq.NamedQubit(s) for s in alphabet)[:n_qubits]
+    qubits = cirq.LineQubit.range(n_qubits)
     swap_network_op = cca.SwapNetworkGate(
         part_lens, acquaintance_size=None)(*qubits)
     swap_network = cirq.Circuit.from_ops(

